data/releasenotes.json

```
@@ -1,5 +1,31 @@
 [
-		{
+	{
+		"version": "v2025-09-24",
+		"date": "2025-09-24",
+		"breaksdb": false,
+		"notes": [
+            "<b>New Feature:</b> CedarDB now allows for persistent configuration parameters via config file. Read more on the <a href= https://cedardb.com/docs/references/configuration/>configuration docs page</a>.",
+			"<b>PostgreSQL compatibility:</b> CedarDB now treats parameter of type <tt>unknown</tt> as <tt>text</tt> like Postgres does.",
+			"<b>PostgreSQL compatibility:</b> The <tt>pg_prepared_statements</tt> system table now also reports the result type of prepared statements.",
```

```suggestion
			"<b>PostgreSQL compatibility:</b> The <tt>pg_prepared_statements</tt> system table now reports the correct result type of prepared statements.",
```
I guess we always reported one, but probably null or so?